### ISSUE TITLE
Ablout DUSt3R baseline evaluate detail

### ISSUE THREAD

**Issue by zhangshuoneu** [Junyi42/monst3r#45]
Table 5 in the paper presents the pose and video estimation accuracy of DUSt3R. I would like to know whether the method mentioned here refers to the result obtained by applying the dynamic pixel region mask mentioned in the paper, or whether it is similar to the Monst3R method, where masking is performed on the output point cloud.
![image](https://github.com/user-attachments/assets/b42ae98e-96f4-4a47-9283-93f03ab92915)

**Comment by Junyi42** [comment]
Hi @zhangshuoneu,

Thanks for pointing this out!

The row you mentioned is for our ablation study on the fine-tuning datasets, and it means use all our optimization technique but only switch the MonST3R model with the **DUSt3R checkpoint**. A more clear notation is "No finetune (DUSt3R ckpt)" rather than "(DUSt3R)".

We will update this in our paper.

Thanks!

**Comment by zhangshuoneu** [comment]
Thanks!
